[Keycloak CI] - AuroraDB IT - Error deleting AuroraDB

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

```
Run gh workflow run aurora-delete.yml \
could not create workflow dispatch event: HTTP 403: Resource not accessible by integration (https://api.github.com/repos/keycloak/keycloak/actions/workflows/99029480/dispatches)
Error: Process completed with exit code 1.
```
https://github.com/keycloak/keycloak/actions/runs/13936023736/job/39003971411

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

_No response_

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -502,6 +502,9 @@ jobs:
     if: needs.conditional.outputs.ci-aurora == 'true'
     runs-on: ubuntu-latest
     timeout-minutes: 150
+    permissions:
+      contents: read
+      actions: write
     steps:
       - uses: actions/checkout@v4
 

